examples/vue/cleanseCards.js

```diff
@@ -0,0 +1,30 @@
+const fs = require("fs");
```

Comment:
Suggestion: remove this file + src/data/cards-raw.json. rename `cards-cleansed.json` to `cards.json`. IMO if they want to ingest, they just need the parsed data documents.

Reply:
This data is constantly changing — Hearthstone keeps getting new cards, and some are getting removed. It doesn't _really_ matter for the demo, but I think if we update it once a year or so, it'll help to keep the demo fresher.